### ISSUE TITLE
Version 1.1.2:  java.lang.IllegalArgumentException: minPort (-1) and maxPort (-1) should be integers between 1024 and 65535.

### ISSUE THREAD

**Issue by naive924** [igniterealtime/openfire-pade-plugin#113]
Version 1.1.2, with the default settings, will definitely fail to join the web conference with the following error.
It seems that it can be avoided with settings `IP Address Mapping > Explicitly not use this mapping. > Disabled`, and I will report it just in case.

* Env
local Windows environment

* Stack trace
```
java.lang.IllegalArgumentException: minPort (-1) and maxPort (-1) should be integers between 1024 and 65535.
    at org.ice4j.ice.harvest.HostCandidateHarvester.checkPorts(HostCandidateHarvester.java:717)
    at org.ice4j.ice.harvest.HostCandidateHarvester.createDatagramSocket(HostCandidateHarvester.java:635)
    at org.ice4j.ice.harvest.HostCandidateHarvester.harvest(HostCandidateHarvester.java:352)
    at org.ice4j.ice.Agent.gatherCandidates(Agent.java:627)
    at org.ice4j.ice.Agent.createComponent(Agent.java:553)
    at org.jitsi.videobridge.transport.ice.IceTransport.<init>(IceTransport.kt:114)
    at org.jitsi.videobridge.transport.ice.IceTransport.<init>(IceTransport.kt:54)
    at org.jitsi.videobridge.transport.ice.IceTransport.<init>(IceTransport.kt)
  ```

* Affected commit
https://github.com/igniterealtime/openfire-pade-plugin/commit/1ea16ca465fe1eb9eecfed0504649c786e156ed8

**Comment by deleolajide** [comment]
> I will report it just in case.

![image](https://user-images.githubusercontent.com/110731/100778228-a5b8e600-33fe-11eb-97c4-01730590503d.png)

Thanks for reporting. It is currently disabled. JVB2 does not appear to expose any parameters for this feature

**Comment by naive924** [comment]
`IP Address Mapping> Explicitly not use this mapping` is the image here.

![image](https://user-images.githubusercontent.com/6185430/100779057-27366580-344b-11eb-8e06-bbe49e45c438.png)

**Comment by deleolajide** [comment]
![image](https://user-images.githubusercontent.com/110731/100780213-5c1dca80-3401-11eb-9f7d-c0292e9ed868.png)

I have experienced this issue already. That is why I disabled the port range for dynamic TCP ports when org.ice4j.ice.harvest.USE_DYNAMIC_HOST_HARVESTER is enabled.  It is not supported in JVB2

IP address mapping is about binding JVB to specific network interfaces and/or IP address like this

![image](https://user-images.githubusercontent.com/110731/100780793-144b7300-3402-11eb-8460-afe2c9a36fe5.png)

This issue will be resolved in version 1.1.3

**Comment by naive924** [comment]
>This issue will be resolved in version 1.1.3
Thank you! I will verify it again.
